### PR TITLE
0.10.0

### LINKED ISSUE #951: release a new version
https://github.com/rustwasm/wasm-pack/pull/839
This fix has been merged for a while, but it's also a year without new release.
Can we draft a new release to fix related issues.

@ashleygwilliams

### PR DESCRIPTION
**Pull request by drager**
Fixes #951

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -44,11 +49,11 @@ jobs:
           echo "using version tag ${GITHUB_REF:10}"
           echo ::set-output name=version::"${GITHUB_REF:10}"
 
-      - name: Install Rust
-        if: matrix.rust
+      - name: Update rust
         run: |
           rustup update ${{ matrix.rust }} --no-self-update
           rustup default ${{ matrix.rust }}
+          rustup target add wasm32-unknown-unknown --toolchain stable
```
Comment (simlay) [inline .github/workflows/release.yml:56]: This shouldn't be needed since you're using the `actions-rs/toolchain` above, right?

**Review by simlay — APPROVED** [review]
Figured I'd do a drive by review (hope that's okay).

Only a minor nit on the `Update rust` step in CI as I think it may be unneeded/redundant.

Looks good otherwise!

I tested this out locally and noticed that when I ran `wasm-pack test --headless --{firefox,safari,chrome}`, I saw:
image

I'm thinking this is because it's not in the installed list wherever it's hosted but I'm not sure.

**Comment by drager** [comment]
> Figured I'd do a drive by review (hope that's okay).
> 
> Only a minor nit on the `Update rust` step in CI as I think it may be unneeded/redundant.
> 
> Looks good otherwise!
> 
> I tested this out locally and noticed that when I ran `wasm-pack test --headless --{firefox,safari,chrome}`, I saw:
> image
> 
> I'm thinking this is because it's not in the installed list wherever it's hosted but I'm not sure.

Absolutely! Appreciate it!

Yeah, I'm pretty sure it's warns now since there's no new release just yet.

File: .github/workflows/release.yml (hunk 1 above)
Comment (drager) [reply .github/workflows/release.yml]: Correct! Nice catch. Thanks!

**Pull request merged**